src/repl.rs

```diff
@@ -351,10 +354,7 @@ impl<F: LurkField> ReplState<F> {
 
                 Ok(())
             }
-            Err(e) => {
-                println!("Evaluation error: {:?}", e);
-                Err(e.into())
-            }
+            Err(e) => Err(e.into()),
```

Comment:
Doesn't this change mean evaluation errors won't be printed when loading/running files? If so, is that really what we intend?

Meta-point: bundling all of these design decisions into one PR intended to fix a specific must-fix language problem make it much harder to review. It would be abstractly better to apply minimal fixes which can be accepted and merged with as little fuss as possible.

I'm not against prospective changes and improvements, but it would be better to separate them into smaller units of work so they can be independently evaluated.

Reply:
Agreed, I bundled this change because it would have been a massive headache to test the necessary changes, without this one. Which at the time seemed trivial enough to include. But if you're right that I broke error messages when loading files, then clearly I was wrong. I'll revert this commit.

Reply:
I see you reverted, but do please make sure you fix the underlying problem (i.e. prevent the REPL from exiting on error). I think adding the  `continue` rather than potentially exiting with `?` was correct and necessary. You just didn't need to move printing the error message.

Reply:
Yes I'm going to move that commit to another branch/PR and work on it there.